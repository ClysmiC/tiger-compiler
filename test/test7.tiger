let
        type int_arr := array [100] of int;
        var a : int_arr := 0;
in
        for i := 2 to 100 do
                a[i] := a[i] + a[i-1];
                printi(a[i]);
        enddo;

        break;

        a + + +
end
