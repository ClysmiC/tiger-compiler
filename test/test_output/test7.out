LET
TYPE
ID
ASSIGN

(Parser error) Line: 2
type int_arr :=<---
Unexpected token found: ASSIGN
Expected token: EQ

ARRAY
LBRACK
INTLIT
RBRACK
OF
INT
SEMI
VAR
ID
COLON
ID
ASSIGN
INTLIT
SEMI
IN
FOR
ID
ASSIGN
INTLIT
TO
INTLIT
DO
ID
LBRACK
ID
RBRACK
ASSIGN
ID
LBRACK
ID
RBRACK
PLUS
ID
LBRACK
ID
MINUS
INTLIT
RBRACK
SEMI
ID
LPAREN
ID
LBRACK
ID
RBRACK
RPAREN
SEMI
ENDDO
SEMI
END
EOF

Unsuccessful parse